site/content/docs/desktop/addons/encode-decode-hash/_index.md

```diff
@@ -0,0 +1,179 @@
+---
+# This page was generated from the add-on.
+title: Advanced Encode / Decode / Hash dialog
+type: userguide
+weight: 1
+cascade:
+  addon:
+    id: encoder
+    version: 0.3.0
+---
+
+# Advanced Encode / Decode / Hash dialog
+
+This allows you to encode, decode or hash text.
+
+## Common Fields
+
+The dialog has one field that is common to all of the tabs:
+
+### Text to be encoded/decoded/hashed:
+
+This field is for the text that you want to be encoded, decoded or hashed.  
+If any text is selected when the dialog is invoked then it will be put in this field. The other fields will be updated dynamically if you change the text  
+
+## Toolbar
+
+* ![](/docs/desktop/addons/encode-decode-hash/images/ui-tab--plus.png) Add New Tab - Adds a new tab to the dialog.
+* ![](/docs/desktop/addons/encode-decode-hash/images/ui-tab--delete.png) Remove Selected Tab - Removes the selected (or current) tab from the dialog.
+* ![](/docs/desktop/addons/encode-decode-hash/images/ui-output--plus.png) Add Output Panel to Current Tab - Adds an output panel to the current tab.
+* Reset - Reset all the tabs/panels to their default state.
+
+## Default Dialog Tabs \& Panels
```

Comment:
Shall I change this to "and" in the help?

Reply:
That's rendered properly.

Reply:
Okay